src/components/footer/footer.scss

```
@@ -0,0 +1,111 @@
+@import "../../components/styles/variables.scss";
+@import "../../components/styles/mixins.scss";
+
+
+.footer {
+  &-content{
+    display: flex;
+    
+  }
+    &-container {
+      height: 492px;
+      width: 100%;
+      background-color: $black-color;
+      
+     
+    }
+    &-bikcraft {
+      text-decoration: none;
+      color: $white-color;
+      font-family: Poppins;
+      font-size: 34px;
+      font-weight: 600;
+      line-height: 72px;
+      letter-spacing: 0em;
+      text-align: left;
+      margin-top: 40px;
+      margin-left: 122px;
+    }
+    &-info-title {
+      font-family: Roboto;
+      font-size: 24px;
+      font-weight: 500;
+      line-height: 32px;
+      letter-spacing: 0.015em;
+      text-align: left;
+      margin-top: 60px;
+      margin-bottom: 32px;
+      margin-left: 300px;
+      text-transform: uppercase;
+    }
+    &-info-ul {
+      li {
+        list-style-type: none;
+        margin-left: -40px;
+      }
+      margin-left: 300px;
+      color: $light-gray-color;
+      font-family: Roboto;
+      font-size: 18px;
+      font-weight: 500;
+      line-height: 32px;
+      letter-spacing: 0.015em;
+      text-align: left;
+    }
```

the first of all write styles for class then u can write other
```suggestion
margin-left: 300px;
      color: $light-gray-color;
      font-family: Roboto;
      font-size: 18px;
      font-weight: 500;
      line-height: 32px;
      letter-spacing: 0.015em;
      text-align: left;
      
      li {
        list-style-type: none;
        margin-left: -40px;
      }
    }
```